src/go/k8s/pkg/console/configmap.go

```diff
@@ -535,7 +535,7 @@ func (cm *ConfigMap) genKafka(username string) config.Kafka {
 			Enabled: y,
 			SchemaRegistry: config.ProtoSchemaRegistry{
 				Enabled:         y,
-				RefreshInterval: time.Second * 10,
+				RefreshInterval: time.Minute * 1,
```

Comment:
Shouldn't the be configurable by user? In other terms exposed to the end user?

Reply:
I have added this to console types to be exposed in the custom resource.